Docker_Windows.md

```diff
@@ -0,0 +1,56 @@
+# Installation under Windows
+## First Run
+1. Navigate to the [Docker download page](https://hub.docker.com/editions/community/docker-ce-desktop-mac/) and download docker.
```

Comment:
I'm actually failing to find an actual download link on the corrected page (though the former link was not much better).
https://docs.docker.com/desktop/windows/install/ may also work.

Reply:
Isn't the big blue "Download for Windows" button working for you? (Maybe you overlooked it because it is not obvious that this is a button.)

If a different link is used here, the screenshot shown below also needs to be replaced.

Reply:
Huh, don't ask me why, but the download buttons were hidden for me. Maybe they're only there when using Windows (running Linux right now)? Weird.